databaseQueries/programUsers.js

```diff
@@ -0,0 +1,57 @@
+module.exports = class programUsers {
+  static programUsersDocument(
+    filterData = "all", 
+    fieldsArray = "all",
+    skipFields = "none"
+  ) {
+    return new Promise(async (resolve, reject) => {
+      try {
+
+        let queryObject = (filterData != "all") ? filterData : {};
+        let projection = {}
+
+        if (fieldsArray != "all") {
+            fieldsArray.forEach(field => {
+                projection[field] = 1;
+           });
+       }
+
+       if( skipFields !== "none" ) {
+           skipFields.forEach(field=>{
+               projection[field] = 0;
+           });
+       }
+
+        let programJoinedData = await database.models.programUsers
+          .find(queryObject, projection)
+          .lean();
+        return resolve(programJoinedData);
+      } catch (error) {
+        return reject(error);
+      }
+    });
+  }
+
+
+  static update(query, update, options = {}) {
```

Comment:
@ankitpws we are keeping update and create operations for program users and read fn in all services.